main.py

```diff
@@ -1,12 +1,226 @@
+        # select video
+        selected_video = video_list.get_video(id=id)
+        print(f"The video you selected is: {selected_video}")
+        title=input('What is the new title? ')
+        release_date=input('What is the new release date? ')
+        total_inventory=int(input('What is new the total inventory? '))
+        response = video_list.update_video(title=title, release_date=release_date,total_inventory=total_inventory)
+        # print output
+        print_decoration()
+        print(f"You updated that video!")
+        print(response)
+        print_decoration()
+
+    elif choice == "10":
+        # delete video
+        print("Okay, lets delete a video!")
+        id = input('What is the id of the video? ')
+            # check if digit. 
+        if id.isnumeric():
+            id = int(id)
+            selected_video = video_list.get_video(id=id)
+            print(f"The video you selected is: {selected_video}")
+
+        response = video_list.delete_video(id=id)
+        print("You deleted that video!")
+
+    elif choice == "11":
+        # checkout a video
+        print("Okay, lets check out a video!")
+
+        # get customer
+        customer_id = input('What is the id of the customer? ')
+        selected_customer = customer_list.get_customer(id=customer_id)
+        print(f"The customer you selected is: {selected_customer}")
+
+        # get video
+        video_id = input('What is the id of the video? ')
+        selected_video = video_list.get_video(id=video_id)
+        print(f"The video you selected is: {selected_video}")
+
+        response = rentals_list.check_out(video_id=video_id,customer_id=customer_id)
+        print(response)
+        
+
+    elif choice == "12":
+        print("Okay, lets check in a video!")
+
+        # get customer
+        customer_id = input('What is the id of the customer? ')
+        selected_customer = customer_list.get_customer(id=customer_id)
+        print(f"The customer you selected is: {selected_customer}")
+
+        # get video
+        video_id = input('What is the id of the video? ')
+        selected_video = video_list.get_video(id=video_id)
+        print(f"The video you selected is: {selected_video}")
+
+        # checkin a video
+        response = rentals_list.check_in(video_id=video_id,customer_id=customer_id)
+        print(response)
+        print(f"You checked in the video {response['video_id']} for the customer {response['customer_id']}")
+
+    elif choice == "13":
+        # Show all options
+        get_actions()
+        pass
 
+    elif choice == "14":
+        # Quit
+        print("Goodbye!")
+        return
+        
+# To be completed:
+    # elif choice == "15":
+    #     customer_id = input('What is the id of the customer? ')
+    #     selected_customer = customer_list.get_customer(id=customer_id)
+    #     print(selected_customer)
+    #     response = selected_customer.get_rentals()
+    #     print(response)
+    #     pass
```

Comment:
The overall user experience would improve if you added a while loop that would run all the logic in main until the user opts out of using the cli via option 14. Currently the CLI completely stops after finishing a task.